ui/apps/pmm-compat/src/lib/utils/navigator.ts

```diff
@@ -0,0 +1 @@
+export const isUserAgentApple = () => /Mac|iPod|iPhone|iPad/.test(navigator.userAgent);
```

Comment:
```suggestion
export const isUserAgentApple = /Mac|iPod|iPhone|iPad/.test(navigator.userAgent);
```
Does it need to be a function?